src/elements/common/annotator-context/__tests__/useAnnotatorEvents.test.tsx

```diff
@@ -80,6 +154,94 @@ describe('src/elements/common/annotator-context/useAnnotatorEvents', () => {
         expect(mockAnnotationUpdateEnd).toBeCalledWith(mockAnnotation);
     });
 
+    test('should call onAnnotationReplyAddStart when proper event is emitted', () => {
+        const mockAnnotationReplyAddStart = jest.fn();
+        const mockEventData = { annotationId: '123', reply: { tagged_message: 'abc' }, requestId: 'comment_456' };
```

Comment:
Seems like annotationID is '123' in every test, you could make that a variable at the top level. Or you can make a mock object which includes the annotationID and just modify the object in each test